designer/client/src/reducers/listReducer.tsx

```diff
@@ -44,6 +49,7 @@ export type ListReducerActions =
   | {
       name:
         | ListActions.EDIT_TITLE
+        | ListActions.EDIT_TYPE
```

Comment:
Thanks for adding this new action

Could you split it out but typed as `ListTypeContent` instead of string?

To make sure we surface errors for non-strings in **ListEdit.tsx**